devtools/maven/src/main/java/io/quarkus/maven/DevMojo.java

```diff
@@ -444,7 +444,7 @@ public void close() throws IOException {
             })) {
                 return scanner.nextLine();
```

Comment:
What about checking if the scanner.hasNextLine() before trying to read the line? It would allow to provide a warn explaining that no prompt was read

Reply:
`scanner.hasNextLine()` seems to block waiting for user input.